client/app/scripts/utils/contrast-utils.js

```diff
@@ -1,7 +1,22 @@
-export const contrastModeUrl = 'contrast.html';
+/* eslint-disable no-underscore-dangle */
+/**
+ * Change the Scope UI theme from normal to high-contrast.
+ * This will inject a stylesheet into <head> and override the styles.
+ *
+ * A window-level variable is written to the .html page during the build process that contains
+ * the filename (and content hash) needed to download the file.
+ */
 
-const contrastMode = window.location.pathname.indexOf(contrastModeUrl) > -1;
 
-export function isContrastMode() {
-  return contrastMode;
+export function loadTheme(theme = 'normal') {
+  if (window.__WEAVE_SCOPE_THEMES) {
+    // Load the pre-built stylesheet.
+    const stylesheet = window.__WEAVE_SCOPE_THEMES[theme];
+    const head = document.querySelector('head');
+    const link = document.createElement('link');
+    link.rel = 'stylesheet';
+    link.href = `${window.__WEAVE_SCOPE_THEMES.publicPath}${stylesheet}`;
+
+    head.appendChild(link);
```

Comment:
This could introduce a race condition where the new sheet isn't loaded before the old one is removed. I guess we could listen for an `onload` event on that `<link />`, but I don't know that the re-paint would be done when that fires?

I don't think there is any harm in leaving that old stylesheet there. Most users will only toggle once.